lib/PN5180/PN5180.cpp

```
@@ -296,37 +296,31 @@ bool PN5180::sendData(uint8_t *data, int len, uint8_t validBits) {
-   */
+  // Idle → Transceive: state machine needs a moment to transition to WaitTransmit
+  writeRegisterWithAndMask(SYSTEM_CONFIG, 0xfffffff8);
+  writeRegisterWithOrMask(SYSTEM_CONFIG, 0x00000003);
+
+  // Stale IRQs from prior commands can cause false failures on the next transaction
+  clearIRQStatus(0xffffffff);
+
+  // Poll for WaitTransmit — transition takes microseconds but isn't instant.
+  // Old code checked once and failed; this matches the NXP reference implementation.
+  unsigned long deadline = millis() + 10;
+  PN5180TransceiveStat transceiveState;
```

Clang (14.0.6)

[warning] variable 'transceiveState' is not initialized

(cppcoreguidelines-init-variables)